app/views/automated_tests/_test_runs.html.erb

```diff
@@ -8,7 +8,8 @@
         assignment_id: <%= @assignment.id %>,
         grouping_id: <%= @grouping.id %>,
         instructor_run: <%= ran_by == 'instructor' %>,
-        instructor_view: <%= !@current_role.student? %>
+        instructor_view: <%= !@current_role.student? %>,
+        rmd_convert_enabled: <%= Rails.application.config.rmd_convert_enabled.to_json %>
```

Comment:
The `.to_json` calls shouldn't be necessary, this option should be a boolean and so should render correctly as just `true` or `false`